error: no matching function for call to ‘google::protobuf::io::CodedInputStream::SetTotalBytesLimit(long unsigned int)’

I see in the commit yesterday that you changed the line from:
`    coded_stream.SetTotalBytesLimit(MessageIterator::MAX_MESSAGE_SIZE * 2, MessageIterator::MAX_MESSAGE_SIZE * 2);`
to 
`     coded_stream.SetTotalBytesLimit(MessageIterator::MAX_MESSAGE_SIZE * 2); `

This generates:
`protobuf_iterator.hpp:274:74: error: no matching function for call to ‘google::protobuf::io::CodedInputStream::SetTotalBytesLimit(long unsigned int)’
     coded_stream.SetTotalBytesLimit(MessageIterator::MAX_MESSAGE_SIZE * 2);
`

Was there a recent update to google protobuf?

--- a/include/vg/io/message_iterator.hpp
+++ b/include/vg/io/message_iterator.hpp
@@ -18,6 +18,14 @@
 
 #include "blocked_gzip_input_stream.hpp"
 
+
+// protobuf scrapped the two-parameter version of this in 3.6.0
+// https://github.com/protocolbuffers/protobuf/blob/v3.6.0/src/google/protobuf/io/coded_stream.h#L387-L391
+// so we hack in support ourselves
+#if (GOOGLE_PROTOBUF_VERSION < 3006000)
+#define SetTotalBytesLimit(N) SetTotalBytesLimit(N, N)
+#endif
+
 namespace vg {
 
 namespace io {

